templates/swift/Sources/Enums/Enum.swift.twig

```diff
@@ -0,0 +1,5 @@
+enum {{ enum.name | caseUcfirst | overrideIdentifier }} : String {
+{% for value in enum.enum %}
+  {% set key = enum.keys is empty ? value : enum.keys[loop.index0] %}
+  case {{ key | caseCamel | replace({'-': ''})}} = '{{ value }}'
```

Comment:
We need to add a comma after every key except the last

Reply:
I couldn't see the format here about adding a comma (https://docs.swift.org/swift-book/documentation/the-swift-programming-language/enumerations/#Raw-Values) . Do you still want me to add it?

Reply:
No need, my mistake